extensions/resteasy-reactive/rest-client-microprofile/runtime/pom.xml

```diff
@@ -0,0 +1,72 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-rest-client-microprofile-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-rest-client-microprofile</artifactId>
+    <name>Quarkus - MicroProfile Rest Client - Runtime</name>
+    <description>MicroProfile Rest Client</description>
+
+    <dependencies>
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-jaxrs-client</artifactId>
+        </dependency>
+        <!--TODO: drop it -->
+        <dependency>
```

Comment:
So this depends on the server? How come?

Reply:
see https://github.com/quarkusio/quarkus/pull/15141#issuecomment-781218934

Reply:
We'll need to look into this more, because we definitely should not depend on the server parts for the client

Reply:
I'll have a look at why the quarkus-jaxrs-client does this and let you know

Reply:
I tried a simple reactive-routes application along with `quarkus-jaxrs-client` and I never ran into the problem you mentioned in the comments.

How can I reproduce the issue?

Reply:
did you use `WebTargetImpl#proxy`?

Reply:
Nope, I'll try it

Reply:
I was able to reproduce it - I'll look into it more

Reply:
we need a client specific implementation of `javax.ws.rs.ext.RuntimeDelegate`, I think